Antti Kokko (1):
      Add changes file for Qt 5.15.2

Heikki Halmet (1):
      Add product_dependencies.yaml to build against LTS 5.15

Jani Heikkinen (2):
      Bump version
      Bump version from 5.15.3 to 5.15.4

Tarja Sundqvist (3):
      Bump version from 5.15.4 to 5.15.5
      Bump version from 5.15.5 to 5.15.6
      Bump version from 5.15.6 to 5.15.7

diff --git a/coin/product_dependencies.yaml b/coin/product_dependencies.yaml
new file mode 100644
index 0000000..3b8e901
--- /dev/null
+++ b/coin/product_dependencies.yaml
@@ -0,0 +1,3 @@
+dependencies:
+  ../tqtc-qt5.git:
+    ref: "tqtc/lts-5.15"
